gum/gumdarwinmodule.c

```diff
@@ -20,6 +20,11 @@
 #define GUM_DARWIN_MODULE_HAS_HEADER_ONLY(self) \
     ((self->flags & GUM_DARWIN_MODULE_FLAGS_HEADER_ONLY) != 0)
 
+#define GUM_SPECIAL_DYLIB_SELF 0
+#define GUM_SPECIAL_DYLIB_MAIN_EXECUTABLE 0xff
+#define GUM_SPECIAL_DYLIB_FLAT_LOOKUP 0xfe
+#define GUM_SPECIAL_DYLIB_WEAK_LOOKUP 0xfd
```

Comment:
I think we should be using `GUM_BIND_SPECIAL_DYLIB_` instead of these, and make sure we interpret the `lib_ordinal` in the chained imports as a signed 8-bit integer. (Unless they use the bits below these as indices, in which case we'll need to do the special-casing in `gum_emit_chained_imports()`.)